src/store/index.js

```diff
@@ -81,7 +81,8 @@ const vuexLocal = new VuexPersistence({
       },
       contacts: state.contacts,
       myProfile: state.myProfile,
-      version: STORE_SCHEMA_VERSION
+      version: STORE_SCHEMA_VERSION,
+      views: state.views,
```

Comment:
We probably don't need to persist this.  Do we want it to re-appear on restart?

Reply:
I suppose we could set it to false on restart so it isn't persisted but it probably belongs in the store unless you think otherwise.  It can be opened currently in many views which makes me think it belongs in the store.